src/Projections/Reunion/Microsoft.UI.Dispatching.DispatcherQueueSynchronizationContext.IDispatcherQueue.cs

```diff
@@ -0,0 +1,38 @@
+#if NET5_0_OR_GREATER
+
+#pragma warning disable CS0649
+
+using System.Runtime.CompilerServices;
+
+namespace Microsoft.System
+{
+    /// <inheritdoc/>
+    public partial class DispatcherQueueSynchronizationContext
+    {
+        /// <summary>
+        /// A struct mapping the native WinRT <c>IDispatcherQueue</c> interface.
+        /// </summary>
+        internal unsafe struct IDispatcherQueue
+        {
+            /// <summary>
+            /// The vtable pointer for the current instance.
+            /// </summary>
+            private readonly void** lpVtbl;
+
+            /// <summary>
+            /// Native API for <see cref="DispatcherQueue.TryEnqueue(DispatcherQueueHandler)"/>.
+            /// </summary>
+            /// <param name="callback">A pointer to an <c>IDispatcherQueueHandler</c> object.</param>
+            /// <param name="result">The result of the operation (the <see cref="bool"/> WinRT retval).</param>
+            /// <returns>The <c>HRESULT</c> for the operation.</returns>
+            /// <remarks>The <paramref name="callback"/> parameter is assumed to be a pointer to an <c>IDispatcherQueueHandler</c> object.</remarks>
+            [MethodImpl(MethodImplOptions.AggressiveInlining)]
+            public int TryEnqueue(void* callback, byte* result)
```

Comment:
>byte [](http://example.com/codeflow?start=50&length=4)

Can this be bool to match the native definition more closely?

Reply:
We can't use `bool*` in the actual function pointer signature, or it will cause issues on .NET Standard 2.0 ([see here](https://github.com/dotnet/coreclr/pull/23899#issuecomment-551371899)). I've changed the parameter to `bool*` though as request, and just changed it to `byte*` internally before invoking the API, which is fine 🙂
Change done in a71a63de4a89435deb2a11ba9e3b9d2859f5307d.